tensorflow/python/ops/ctc_ops.py

```diff
@@ -180,8 +180,8 @@ def ctc_greedy_decoder(inputs, sequence_length, merge_repeated=True):
   only the first of these is emitted.  The sequence `A B B * B * B` (where '*'
   is the blank label) becomes
 
-    * `A B` if `merge_repeated=True`.
-    * `A B B B B B` if `merge_repeated=False`.
+    * `A B B B` if `merge_repeated=True`.
```

Comment:
This seems at odds with the unit test [here](https://github.com/petewarden/tensorflow_ios/blob/master/tensorflow/contrib/ctc/ctc_decoder_ops_test.py#L132), no?